src/status/libstatus/wallet.nim

```diff
@@ -32,6 +33,36 @@ proc getWalletAccounts*(): seq[WalletAccount] =
     let msg = getCurrentExceptionMsg()
     error "Failed getting wallet accounts", msg
 
+proc getTransfersByAddress*(address: string): seq[Transaction] =
+  try:
+    var response = status.getBlockByNumber("latest")
+    let latestBlock = parseJson(response)["result"]
+    
+    var transactionsResponse = status.getTransfersByAddress(address, latestBlock["number"].getStr, "0x14")
+    let transactions = parseJson(transactionsResponse)["result"]
+    var accountTransactions: seq[Transaction] = @[]
+
+    for transaction in transactions:
+      accountTransactions.add(Transaction(
+        typeValue: transaction["type"].getStr,
+        address: transaction["address"].getStr,
+        blockNumber: transaction["blockNumber"].getStr,
+        timestamp: transaction["timestamp"].getStr,
+        gasPrice: transaction["gasPrice"].getStr,
+        gasLimit: transaction["gasLimit"].getStr,
+        gasUsed: transaction["gasUsed"].getStr,
+        nonce: transaction["nonce"].getStr,
+        txStatus: transaction["txStatus"].getStr,
+        value: transaction["value"].getStr,
+        fromAddress: transaction["from"].getStr,
+        to: transaction["to"].getStr
```

Comment:
All the values here are their raw hexadecimal counterparts. Probably makes sense to have some of them converted here.

Reply:
Yeah, or in a `toTransaction` function